drivers/i2c/Kconfig

```diff
@@ -22,6 +22,12 @@ config I2C_SHELL
 	  The I2C shell supports scanning, bus recovery, I2C read and write
 	  operations.
 
+config I2C_ZERO_LEN_XFER_SUPPORTED
+	bool "I2C transfer without data (data length 0)"
+	default y
```

Comment:
Shouldn't this be disabled by default and selected by drivers supporting zero-length transfers?